First deploy

IshmaelKargbo opened the pull request:
Please review my code and give your comment or recommendation on the requested changes:
1. I replace the author name with my name
2. I added a description of the project
3. I added a gitignore file and ignore the test.md and also all the node_modules

Thanks 😄

safafa commented on README.md:
```diff
@@ -1 +1,42 @@
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```
- [ ] [OPTIONAL] I think it's better to remove this line or modify it by adding an additional description about the project to make your REadme more professional.

safafa reviewed (changes requested):
## STATUS: CHANGES REQUESTED ♻️ 

Hi @IshmaelKargbo, You did a great job so far 👏🏼 👏🏼 

- The linters are well set and all the checks are passing.
- The github-flow is well set
- Well done on using Microvers readme template.

However, here are some points that you need to work on to get your project approved.

## Required changes :

Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Commiting more often after every significant change you make. Having a professional commit history will help you better track your project. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

IshmaelKargbo commented:
@safafa please review the requested changes:
1. I replace the author name with my name
2. I added a description of the project
3. I added a gitignore file and ignore the test.md and also all the node_modules
Thanks 😄

DanteAlonsoHT commented on test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
You did a good job adding this file in your `.gitignore` file, but also, don't forget to remove this file from your repository due to for this reason you are adding this file in your `.gitignore` file to don't have this file in your repository 😉 

- [ ] Just remove/delete this file from your repository as the project requirement expect when you add this file in your `gitignore`😉 you should have `test.md` in your local repository, but not in your GitHub repository.

DanteAlonsoHT reviewed (changes requested):
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @IshmaelKargbo,, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

DanteAlonsoHT reviewed (approved):
<h1>Status: Approve 🥇 ✔</h1>
Hi @IshmaelKargbo,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

<img src="https://media.giphy.com/media/aNqEFrYVnsS52/giphy.gif" width="100%">

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
